website/src/pages/contributing/contributors/ncContributors.tsx

```diff
@@ -0,0 +1,29 @@
+export interface Contributor {
```

Comment:
This is the code contributor interface, if you have a look at the issue, the none code contributor one is a bit different.